package.json

```diff
@@ -39,12 +39,13 @@
   "scripts": {
     "check": "gts check",
     "clean": "gts clean",
-    "compile": "tsc -p .",
+    "build": "tsc",
```

Comment:
Seems to be slightly more common than `compile`.

Reply:
I'm not at all religious about this :) but I've used Google's `gts` configurator here in order to have some agreement on naming things. `gts` uses `compile` but also `build` instead of `dist` (https://github.com/google/gts).

Reply:
Good point; reverted my change from `dist` to `build`.